python/simulators/CMakeLists.txt

```diff
@@ -67,3 +60,22 @@ if(OPM_ENABLE_PYTHON_TESTS)
       -E env PYTHONPATH=${PYTHON_PATH} ${PYTHON_EXECUTABLE}
       -m unittest test/test_schedule.py)
 endif()
+
+# NOTE: instead of using file( COPY ...) which copies the files at configure time (not at build time)
+#  we should add copying of the files at build time such that running "make" or "make all" will
+#  update the files if they have been modified. We use the install.py script in opm-common, see also
+#  CMakeLists.txt in opm-common
+add_custom_target(copy_python ALL
+    COMMAND ${PYTHON_EXECUTABLE} ${opm-common_DIR}/python/install.py
+      ${PROJECT_SOURCE_DIR}/python ${PROJECT_BINARY_DIR} 0)
+
```

Comment:
> Are you sure that this works for an installed opm-common? 

@blattms Thanks for the comment. I always use this folder structure when  building `opm-simulators`: The other opm-modules are located in an immediate parent folder. For example, the location of the opm-common git repository is `../opm-common` relative to `opm-simulators`. Could you share how do you build against an "installed opm-common" ? Do we need to consider other options for building `opm-simulators` ? 

>  instead of the obscure python script

My idea was to avoid duplicating code in `opm-common`, see [line 413](https://github.com/OPM/opm-common/blob/82a5e7192e4ff621ca3ee5098a7d5d3a2f9d54de/CMakeLists.txt#L413) It seemed like a good idea since it also avoided listing all the files to copy (in the future the number of files may increase) and the python script searches and creates sub folders as needed. What do you think?

Reply:
Yes sibling builds is only one way of building, jenkins for instance does not use it. You just do like you do when you build any piece of software on linux, you build opm-common and call install from there, rinse repeat for every module.

Reply:
@hakonhagland take a look at OPM/opm-common#3034 on how to do this with Cmake directly. I do not think doing that here is duplicating code